src/main/java/org/opensearch/performanceanalyzer/rca/store/OpenSearchAnalysisGraph.java

```diff
@@ -463,8 +464,17 @@ public void construct() {
                 RcaConsts.RcaTagConstants.TAG_AGGREGATE_UPSTREAM,
                 RcaConsts.RcaTagConstants.LOCUS_DATA_NODE);
 
-        // TODO: Add SearchBackPressure Decider
+        // SearchBackPressure Decider
```

Comment:
Let us add all the SearchBackPressure node in the Analysis graph behind a feature flag.